src/odemis/gui/cont/tabs.py

```diff
@@ -498,6 +500,17 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 self
             )
 
+    def _onView(self, view):
+        """Hide hardware related sub-panels on the right when not in live stream"""
+        if "Live" in view.name.value:
```

Comment:
In general, it's best to not depend on text for user. If one day we decide to adjust this text (ex: call it "Acquisition" instead of "Live"), suddenly this feature will stop working. So, I'd rather have the check rely on the view.stream_classes.  as we discussed previously. I think you can do something like `if is_subclass (view.stream_classes, LiveStream)`.  Or are there any reason not to?